openquakeplatform/openquakeplatform/vulnerability/views.py

```diff
@@ -986,7 +995,10 @@ def _stream_frag_discr_data(curr, func_obj, is_first):
         desc.text = 'Fragility from GVD: %s' % func_obj.name
 
         lims = etree.SubElement(frmo, "limitStates")
-        lims.text = func_obj.fragility_func.limit_states_desc.replace(";", " ")
+        lims.text = re.sub(
```

Comment:
Please add a comment like what you have added to the changelog (here and below), and perhaps extract this into a separate function, to avoid duplication and to improve readability.